test-cases/performance/perf-regression-gradual-throughput-grow.yaml

```
@@ -1,34 +1,64 @@
+  "cassandra-stress read no-warmup cl=QUORUM duration=30m -mode connectionsPerHost=8 cql3 native -rate 'threads=$threads $throttle' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=10000001..15000000",
+  "cassandra-stress read no-warmup cl=QUORUM duration=30m -mode connectionsPerHost=8 cql3 native -rate 'threads=$threads $throttle' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=15000001..20000000"
+]
+#  ["cassandra-stress read  cl=QUORUM duration=30m -mode cql3 native -rate 'threads=$threads $throttle' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..15000000",]
 
+stress_cmd_m: [
+  "cassandra-stress mixed no-warmup cl=QUORUM duration=30m -mode connectionsPerHost=8 cql3 native -rate 'threads=$threads $throttle' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..5000000",
+  "cassandra-stress mixed no-warmup cl=QUORUM duration=30m -mode connectionsPerHost=8 cql3 native -rate 'threads=$threads $throttle' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=5000001..10000000",
+  "cassandra-stress mixed no-warmup cl=QUORUM duration=30m -mode connectionsPerHost=8 cql3 native -rate 'threads=$threads $throttle' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=10000001..15000000",
+  "cassandra-stress mixed no-warmup cl=QUORUM duration=30m -mode connectionsPerHost=8 cql3 native -rate 'threads=$threads $throttle' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=15000001..20000000"
+]
+#  ["cassandra-stress mixed  cl=QUORUM duration=30m -mode cql3 native -rate 'threads=$threads $throttle' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..15000000",]
 
-prepare_write_cmd: "cassandra-stress write no-warmup cl=ALL n=$ROW_NUMBER -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=$SEQUENCE"
-
-stress_cmd_m: ["cassandra-stress mixed 'ratio(write=1,read=1)' no-warmup cl=QUORUM duration=30m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=$threads throttle=$throttle/s' -pop 'dist=$DIST_PARAMS' "]
-stress_cmd_w: ["cassandra-stress write no-warmup cl=QUORUM duration=30m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=$threads throttle=$throttle/s' -pop 'dist=$DIST_PARAMS' "]
-stress_cmd_r: ["cassandra-stress read no-warmup cl=QUORUM duration=30m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=$threads throttle=$throttle/s' -pop 'dist=$DIST_PARAMS' "]
-
-compaction_strategy: "LeveledCompactionStrategy"
-
+round_robin: true
 
 n_db_nodes: 3
-n_loaders: 3
+n_loaders: 4
 n_monitor_nodes: 1
 
-instance_type_loader: 'c5n.9xlarge'
-instance_type_monitor: 't3.large'
-instance_type_db: 'i3.4xlarge'
 
-user_prefix: 'perf-1tb-scylla-gradual-grow-throughput'
-space_node_threshold: 644245094
-ami_id_db_scylla_desc: 'VERSION_DESC'
+instance_type_db: 'i4i.4xlarge'
+instance_type_loader: 'c7i.12xlarge'
```

```suggestion
instance_type_loader: 'c7i.8xlarge'
```

Didn't test, but should be ok.